geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/sortedset/AbstractSortedSetRangeOptions.java

```diff
@@ -66,26 +66,31 @@ void parseLimitArguments(List<byte[]> commandElements, int commandIndex) {
 
   // If limit specified but count is zero, or min > max, or min == max and either are exclusive, the
   // range cannot contain any elements
-  boolean isEmptyRange() {
-    int minVsMax = compareMinToMax();
-    return (hasLimit && (count == 0 || offset < 0)) || minVsMax == 1
-        || (minVsMax == 0 && (isMinExclusive || isMaxExclusive));
+  boolean isEmptyRange(boolean isRev) {
+    int startVsEnd = compareStartToEnd();
```

Comment:
Much is now repeated between the if and then else.
I think this code would be clearer if you didn't try to do everything in one boolean expression. For example:
if (hasLimit && (count == 0 || offset < 0)) {
  return true;
}
int startVsEnd = compareStartToEnd();
if (startVsEnd == 0 && (isStartExclusive || isEndExclusive)) {
  return true;
}
if (isRev) {
  return startVsEnd == -1;
} else {
  return startVsEnd == 1;
}

Reply:
Adjusted the code, and the leading comment to match the test order.